assets/dev/scss/editor/navigator/navigator.scss

```diff
@@ -113,9 +110,10 @@ body {
 		&__promotion {
 			font-size: 12px;
 			padding: 4px 17px;
-			line-height: 22px;
 			font-family: 'Roboto', sans-serif;
 			color: var(--e-a-bg-logo);
+			min-height: 30px;
+			line-height: 18px;
```

Comment:
Why has the line height changed?

Reply:
We try to keep the number of changes in a PR as low as possible, to make the review easier.

Reply:
@Batsirai-muchareva Can we rather use `display: flex;` with `align-items: center;`? Not sure if line-height was the best approach to begin with for controlling the alignment.

Reply:
I know Hadar probably approved it, but to me, the new screenshots look even less vertically aligned than before 🤔 WDYT?